lib/indico.rb

```diff
@@ -26,6 +26,9 @@ def self.private_cloud=(cloud)
   end
 
   def self.political(text, config = nil)
+    unless !config or config.key?(:v) or config.key?(:version)
+      config['version'] = "2"
```

Comment:
Yeah, wait.  I don't think this works unless config is already set to something?  The `!config` should mean that config['version'] never gets set.